packages/client/src/__tests__/migrations.test.ts

```diff
@@ -305,4 +306,41 @@ describe("Migrations", () => {
       ).rejects.toThrow(HTTPSErrors.Unauth);
     });
   });
+  describe("default email templates population", () => {
+    testWithEmulator(
+      "should populate default templates in existing organizations",
+      async () => {
+        const { organization } = await setUpOrganization();
+
+        const orgRef = adminDb.doc(`${Collection.Organizations}/org1`);
+
+        await orgRef.set({}),
+          await invokeFunction(CloudFunction.PopulateDefaultEmailTemplates)({
+            organization,
+          });
```

Comment:
Can we just remove this test...out CI server is overworked anyway, the test is failing and is not really necessary